torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -555,4 +555,13 @@ TEST(ProcessGroupGlooTest, testBroadcastCUDA) {
   }
 }
 
+TEST(ProcessGroupGlooTest, testAlltoallCUDA) {
+  {
+    if (torch::cuda::is_available()) {
+      TemporaryFile file;
+      testBroadcast(file.path, at::DeviceType::CUDA);
```

Comment:
Am I missing something or should this test the `allToall` functionality?

Reply:
oops published from a different dev box